internal/core/Cargo.toml

```diff
@@ -83,6 +83,10 @@ resvg = { version= "0.23", optional = true, default-features = false }
 usvg = { version= "0.23", optional = true, default-features = false, features = ["text"] }
 tiny-skia = { version= "0.6", optional = true, default-features = false }
 
+# This is sub-optimal, it should be limited to esp32 family, but sadly rustc --print=cfg doesn't show any variables to narrow this choice.
+[target.'cfg(target_arch = "xtensa")'.dependencies]
+esp-println = { version = "0.3.0", optional = true, default-features = false }
```

Comment:
what if you don't make it optional, but always enable it on the xtensa target? then you don't need the feature.

Reply:
I tried that first, but that breaks the `--all-features` build :-(

Reply:
interresting,  why does it build the --all-features build.  I would expect that even with  --all-features, it wouldn't be enabled.